### ISSUE TITLE
Oh my word Chef sucks (Issue summup)

### ISSUE THREAD

**Issue by asperger-sind** [space-wizards/space-station-14#9313]
## The tl;dr:

Playing Chef right now sucks, the role has essentially no interactions except for the one with Botany which starts and ends when you say what you need and what they do, half the recipes aren't implemented into the game, half the recipes that are implemented can't be made as the ingredients for them can't be made either. 
Cooking itself sucks, you point and click for the entire shift, no danger, no pressure from game mechanics (e.g food burning), no interesting or unique cooking schemes, sauces are *in* but nobody uses them and can't exactly use them either.
Microwaves are the dumbest thing ever, Chef also lacks any unique personal gimmicks service roles tend to have.
You can play the entire shift, infact, several shifts in a row without leaving your department for absolutely anything and not be punished, nor are you encouraged to do so besides the usual cargo run or med if you somehow got injured, presumably by touching lights, and this issue, ends up in such a way that even if the crew is having 40 antags, nukies and what not, as long as the Chef is in the Kitchen with power and ingredients nothing changes, the gameplay is the same, nobody touches him and he just makes food.

## Interactions.
So, you supposedly should have interactions with 3 sides, the ones you feed, Botany, and med with the morgue, sometimes Cargo, but Cargo is more or so an universal interaction to every department/job.

**Botany**
This *should* work in practice and occasionally works, but the current interaction most Chefs and Botanists have are
>"Hey, Botany, make me X ingredient, also Y ingredient, and Z ingredient, I'll make some Chili."
>"Okay, Chef, will do."

And then the Botanist proceeds to make said ingredients, and the Chef proceeds to make Chili, even if the Chef tries to make new food with brand new ingredients he doesn't have it's still the same 1 line social interaction with Botany, said Botany which occasionally asks to dry some weed, that's it, it ends there on practice, in theory it works in practice it's barely encouraged.

**Med**
Everyone knows that the Chef loooves the bodies from morgue, which not only barely makes sense that you can use rotten flesh as an ingredient for a fresh burger with no side effects, but now, you don't even need to come there, with Miasma added people just bring you the corpses themselves and this albeit already boring as shit interaction ends up getting nulled to saying "Thanks" after you get your free non-monkey meat.

**Clients**
This is an interesting one, it's player dependent, and most likely'll get much, much better with MRP, but as it stands, they come to the counter, you toss food, they say "Thanks", wooow! Or better even, you just leave the food on the counter as most Chefs do and even this ends up being nullified.

**Interactions conclusion**
Chef has some in theory, they're minor, and don't even work in practice, I think the Chef should have drawbacks from using rotten bodies, so Chem has to sterilize them with some chemical beforehand, Botany and the Chef constantly need to have some sort of driving force to force them away from making the same food over and over, say, a system of being fed up with certain food and having to eat brand new food you haven't eaten before to encourage this interaction, even if it's minor, or, make Botany somewhat dependent on the Chef as the Chef is dependent on them.

## Cooking itself
Oh my word, that's a scuffed one.

**Recipes, anyone?**
Yeah, you know what this is, no custom recipes whatsoever, half the recipes that have sprites added for them can't be made, half the ingredients can't be used, the other half can't be made, we've been over this and this is a major wyci.

**Cooking mechanics**
Obliterate the microwave, no, I can't see this again.
We need proper utensils and an actual frying/boiling/baking, etc system along with the fact that you can burn food, that'll make Chef at least more complex than he is now, because stuffing dough into a microwave for 15 seconds to bake it makes sense.
The Chef also needs a way to bulk-produce food, as the Crew is hungry and the Crew is large, it only makes sense if you can bake a large loaf of bread or heaps of loafs at the same time instead of baking 1 bread loaf per 15 seconds.
As a sidenote, current cooking is EXTRAORDINARILY point and click reliant, seriously, after playing Chef for an hour long shift my wrist hurts incredibly and I just can't keep playing anything, this is probably a pretty good indicator your job isn't exactly good.
**No seriously it's worse than playing chem and mass producing**

**Sauces, spices, custom recipes, oh god nobody's gonna code this oh shit**
Yeaaaah, uhm, uhhh, wyci.
Really, nobody's gonna code this, ever.

**Clonclushmon**
Cooking boring, cooking stinky, cooking microwave placeholder 1984 when.
~~notice how this becomes more of a shitpost by the line, yeah me neither~~

## Unique stuffs
The Chef lacks, really lacks, for example, 13 Botany, the quite literal ingredient slaves of the Chef could do so much, it was one of the more vibrant sandboxxy departments, and now we have this...

**Not really sure, need suggestions, NO NOT CQC PLEASE NO**
Github comments my favorite part of having this issue closed 2 minutes in because of course aaaaaaaaaaaaaaaaaaaaaaaa

**Additional context**

God this part of the Crew needs some love.

**Comment by ShadowCommander** [comment]
Chef is one of the things that hasn't actually been touched in a long time because it's waiting on someone to come along and tweak construction graphs to be able to handle nearby machines (in this case ovens, fryers, etc.). If you'd like to work on it then feel free. If you need help coding or learning how to code, then ask in the #howdoicode channel on Discord.

We have a goal for what cooking/chef should be here https://docs.spacestation14.io/en/design/kitchen.

Since this issue isn't really something actionable, I'm going to close it, but if you want you can make an issue following the doc with checkboxes for what should be implemented then feel free.
